common/changes/@adobe/ccweb-add-on-analytics/release-20240525_2024-05-25-16-21.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@adobe/ccweb-add-on-analytics",
+      "comment": "Updates for tiktop API",
```

Comment:
Typo. Besides you may update the comment to:
```
Add TikTok CML as a trusted partner in manifest.json.
```
Do this on all the change files.